openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -88,6 +88,7 @@ def to_representation(self, user):
             "level_of_education": AccountLegacyProfileSerializer.convert_empty_to_None(profile.level_of_education),
             "mailing_address": profile.mailing_address,
             "requires_parental_consent": profile.requires_parental_consent(),
+            "account_privacy": UserPreference.get_value(user, 'account_privacy'),
```

Comment:
It will be the same number of queries either way.  Either queries += 1 for the explicit query, or += 1 for a prefetch (if we do that). The advantage of the prefetch is that if we later add more preferences, we can take advantage of the single query to get all the preferences.  The disadvantage is that you can only use `.all()`.  You can't add more filters to the query.  But there's another, more straightforward way to do it, which would be `user.preferences.all()` (or `UserPreferences.filter_by(user=user).all()`) at the top of the query.  

`prefetch_related()` only really starts to shine when you're trying to get preferences for multiple users.  Then you can do something like:

```python
    users = User.objects.prefetch_related('preferences').all()
    for user in users:
         print user.preferences.all()
```

...and still only use two queries, no matter how many users you have.